spec/machine/ARM/Platform.thy

```diff
@@ -76,9 +95,23 @@ definition cacheLineBits :: nat where
 definition cacheLine :: nat where
   "cacheLine = 2^cacheLineBits"
 
-(* The first virtual address of the kernel's physical memory window *)
+(* The first virtual address of the kernel's physical memory window.
+   Coincides with seL4_UserTop and USER_TOP in C for AArch32.
+
+   The definition below singles out the omap3 platform, because it is the only AArch32
+   platform that does not put seL4_UserTop at 0xe0000000. The macro definition in C is
+   hard to get at for the kernel config generation script and the definition here is
+   comparatively straightforward. *)
```

Comment:
a potential concern: if this changes on the C side, will we pick them up? I'm guessing only during CRefine, which would be ok since we don't expect these to change.